README.md

```diff
@@ -252,6 +257,26 @@ To build LPVS from source code and run it, follow these steps:
 
 LPVS is now built and running. You can create a new pull request or update an existing one with commits, and LPVS will automatically start scanning and provide comments about the licenses found in the project.
 
+#### 4.b Single scan mode
+
+Another configuration is available to run a one-time scan on a single pull request.
+
+   To access it first you run installation and navigate to the target directory the same way you do for the service mode (following steps 4.1 and 4.2):
+   ```bash
+   mvn clean install
+   cd target/
+   ```
+
+   Then you can run a single scan using the following command:
+   ```bash
+   java -jar -Dgithub.token=<my-token> lpvs-*.jar --github.pull.request=<PR URL>
+   ```
+
+   In the case of the previous command you still need to set up MySQL database beforehand for the program to work, which can be inconvenient. In order to not have to set up the database you should choose "singlescan" profile, like this:
+   ```bash
+   java -jar -Dspring.profiles.active=singlescan -Dgithub.token=<my-token> lpvs-*.jar --github.pull-request=<PR URL>
+   ```
+
```

Comment:
#### 4.b Single scan mode

Alternatively, you can perform a one-time scan on a specific pull request using the single scan mode. Follow these steps:

4.b.1. Begin by running the installation and navigating to the target directory, similar to the process in service mode (refer to steps 4.a.1 and 4.a.2):
```bash
   mvn clean install
   cd target/
   ```

4.b.2. Execute the single scan with the following command:
```bash
   java -jar -Dgithub.token=<my-token> lpvs-*.jar --github.pull.request=<PR URL>
   ```

4.b.3. By default, the above command requires a pre-configured MySQL database. To avoid setting up the database, use the "singlescan" profile:
 ```bash
   java -jar -Dspring.profiles.active=singlescan -Dgithub.token=<my-token> lpvs-*.jar --github.pull-request=<PR URL>
   ```
These steps streamline the process, allowing you to run a scan on a single pull request without the need for a preconfigured database.